### ISSUE TITLE
Save fit parameters to a file

### ISSUE THREAD

**Issue by will-henney** [div-B-equals-0/confitti#3]
It would be useful to be able to serialize the results from the fits so that we can use them afterwards. I should work out what the simplest way of doing this is.

**Comment by will-henney** [comment]
I can use ASDF for this, which is like a cross between YAML and FITS. 

- [Docs of asdf python package](https://asdf.readthedocs.io/en/latest/index.html#asdf)
- Also, see [asdf-astropy](https://asdf-astropy.readthedocs.io/en/latest/index.html#asdf-astropy)
- And the [documentation of the ASDF format](https://asdf-standard.readthedocs.io/en/latest/)

**Comment by will-henney** [comment]
So it turns out that ASDF cannot be used out of the box, since it uses yaml.safe_dump under the hood, which does not work on arbitrary python objects. I would have to write a special Converter for it, which looks like quite a lot of work. 

Therefore, I am looking into other options:

1. Just use pickle on the MinimizerResult object. This may or may not work. And it would mean that the save file can only be read by the exact same python version that was used to create it. 
2. JSON. This might be better, since the Parameters class (which is the most important component of MinimizerResult) has .dumps() and .loads() methods that convert to/from json representations. 
3. Alternatively (or additionally) we could just save the XYConic object instead of the lmfit result. This might be enough for Vanessa's purposes.

**Comment by will-henney** [comment]
I have now decided not to try and serialize the lmfit result, but just to save the parameters and their uncertainties

**Comment by will-henney** [comment]
That seems to work at least partially. For example

```yaml
params:
  eccentricity: 1.0830661105211852
  r0: 7.5053539040889685
  theta0: 162.31801146800825
  x0: -0.32146936319570624
  y0: -1.3853933155243527
uparams:
  eccentricity: 0.05258668094648568
  r0: 0.5690418171170513
  theta0: 1.4801671974934552
  x0: 0.5755166597405751
  y0: 0.5344204040883483
```

But I think I need to save the dat points as well

**Comment by will-henney** [comment]
Fixed in b0650bf42018c8bf2a693fafe371a279d7d2c8d0
